nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -305,6 +305,14 @@ in {
       example = "php82";
     };
 
+    packageWithApps = mkOption {
+      type = types.path;
+      readOnly = true;
+      description = ''
```

Comment:
I think a `defaultText` would be nice here to indicate that this is already set by the module.

Reply:
hmm I am confused about this request. there is no default defined, so why should i add the `defaultText`?

Maybe you just wanted to add this to the description?

Reply:
I mean, the module is effectively setting a default, but I think the description is also sufficient :shrug: